spec/support/new_supplier_helper.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module NewSupplierHelper
+  # rubocop:disable Naming/MethodName
```

Comment:
Does it dislike capital `I`? length? Anything else?

Reply:
The 'I'